swagger-macro/src/main/resources/Macros/Swagger Translations.xml

```diff
@@ -0,0 +1,112 @@
+  <name>Swagger Translations</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>Swagger Translations</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>plain/1.0</syntaxId>
+  <hidden>true</hidden>
+  <content>rendering.macro.swagger.description=Enables you to display OpenAPI specifications directly in your wiki.
+rendering.macro.swagger.name=Swagger
+rendering.macro.swagger.parameter.defaultModelRendering.description=Example JSON (example), JSON schema (model)
+rendering.macro.swagger.parameter.defaultModelRendering.name=Show model as
+rendering.macro.swagger.parameter.displayOperationId.description=Show the ID of operations
+rendering.macro.swagger.parameter.displayOperationId.name=Display Operation ID
+rendering.macro.swagger.parameter.displayRequestDuration.description=Show the duration of Try it out requests
+rendering.macro.swagger.parameter.displayRequestDuration.name=Show request duration
+rendering.macro.swagger.parameter.filter.description=Show only this tag
+rendering.macro.swagger.parameter.filter.name=Filter
+rendering.macro.swagger.parameter.maxDisplayedTags.description=Show at most this many tags
+rendering.macro.swagger.parameter.maxDisplayedTags.name=Limit number of tags displayed
+rendering.macro.swagger.parameter.operationsSorter.description=Definition order (default), path (alpha), HTTP method (method)
+rendering.macro.swagger.parameter.operationsSorter.name=Methods sorter
+rendering.macro.swagger.parameter.password.description=Optional credentials for accessing the above URL
+rendering.macro.swagger.parameter.password.name=Password
+rendering.macro.swagger.parameter.proxy.description=Move the requst to retrive the OpenAPI specification from frontend to the backend to avoid CORS issues.
+rendering.macro.swagger.parameter.proxy.name=Cors Proxy
+rendering.macro.swagger.parameter.showCommonExtensions.description=Show extensions (pattern, maxLength, minLength, maximum, minimum) fields and values for Parameters
+rendering.macro.swagger.parameter.showCommonExtensions.name=Show common extensions
+rendering.macro.swagger.parameter.showExtensions.description=Show x- fields and values for Operations/Parameters/Schema
+rendering.macro.swagger.parameter.showExtensions.name=Show vendor extensions
+rendering.macro.swagger.parameter.supportedSubmitMethods.description=List the operations allowed in 'Try it out', separated by commas: get, post, put, delete, head, options, patch, trace. To disable 'Try it out' completely, enter none
+rendering.macro.swagger.parameter.supportedSubmitMethods.name=Supported submit methods
+rendering.macro.swagger.parameter.tagsSorter.description=Definition order (default), name (alpha)
+rendering.macro.swagger.parameter.tagsSorter.name=Tags Sorter
```

Comment:
You also need to add some translation keys for the enumaration values. The prefix seems to be the class type:  https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/html_displayer/enum/edit.vm#L31-L32

As an example: https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsTranslations.xml#L49-L50